victoria-hedenmalm/assignment-1/assignment1_extra.py

```
@@ -0,0 +1,40 @@
+from collections import defaultdict
+
+def word_anagram(first, second, case_sensitive=False, only_allow_alpha=False):
+    if not only_allow_alpha:
```

Shouldn't there be no 'not' here as we would want to throw the exception only when 'only_allow_alpha' is enabled?